soh/src/code/z_eff_blure.c

```diff
@@ -73,6 +75,11 @@ void EffectBlure_AddVertex(EffectBlure* this, Vec3f* p1, Vec3f* p2) {
     }
 }
 
+//dumb doo doo command to change the type of an object's blur on the fly. Link's Swords with unique trail colors.
```

Comment:
this is perfectly reasonable, just adding a method because sometimes the type needs to be changed outside of `z_eff_blure`

Reply:
Yeah, it does with swords, otherwise changing settings mid-swing with bug out a bit. That, and if it set with other things it would just be way messier. I'm surprised I called this the dumb doo doo code instead of the massive bloat setting the proper colors for everything.